extensions/opentelemetry/opentelemetry/runtime/src/test/java/io/quarkus/opentelemetry/runtime/tracing/TracerUtilTest.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.opentelemetry.runtime.tracing;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
+
+public class TracerUtilTest {
+
+    @Test
+    public void testMapResourceAttributes() {
```

Comment:
Unit test on `TracerUtil.mapResourceAttributes()` to ensure the refactoring didn't introduce a regression.